packages/jest-resolve/package.json

```diff
@@ -9,9 +9,10 @@
   "main": "build/index.js",
   "dependencies": {
     "browser-resolve": "^1.11.2",
-    "chalk": "^2.0.1"
+    "chalk": "^2.0.1",
+    "jest-util": "^21.2.1"
   },
   "devDependencies": {
     "jest-haste-map": "^21.2.0"
   }
-}
+}
```

Comment:
Please bring back this newline. I was always very fond of it and would hate to see it fall victim to your change.